### ISSUE TITLE
Is this the full code? Not running

### ISSUE THREAD

**Issue by DIN05AUR** [UNICT-Fake-Audio/fake-audio-detector#5]
Does this repo contain full length code? Or it is just the part of a code, because there are just few files (just 3 .py files) and when I tried running it, it shows error FILE_PATH = sys.argv[1]
RuntimeWarning: Couldn't find ffmpeg or avconv..........................IndexError: list index out of range

**Comment by Helias** [comment]
`RuntimeWarning: Couldn't find ffmpeg or avconv...` some libraries could use ffmpeg, be sure to have it installed.

Moreover, try to give as input a sample with only one channel
`ffmpeg -i input.wav -ac 1 output.wav`

**Comment by sidorov-alex** [comment]
yeah it's fake fake detection tool. it's only print Python and no more. fuck you in your deep ass, author
